crates/codegen/parser_templates/src/rust/cst.rs

```diff
@@ -0,0 +1 @@
+../shared/cst.rs
```

Comment:
are these symlinks? not sure I understand how they are used, since `shared/cst.rs` is loaded directly in codegen.

Reply:
If the only purpose is to include them in this folder's `mod.rs`, is it possible to just use a path rewrite there instead?

```rust
// In mod.rs:

#[path = "../shared/foo.rs"]
mod foo;
```

Reply:
Yes, they are symlinks in order that they can be shared between the languages whilst other files can refer to them using paths that are the same as when they are used. The templates projects are 'live' in the sense that they are valid, and work with intellisense, analyzer etc.

Reply:
I guess a path rewrite would be ok, as long as analyzer and intellisense handle that correctly

Reply:
Please see `crates/codegen/schema/build.rs` as an example. It is working well with Rust Analyzer and intellisense.